docs/developing-locally-docker.rst

```diff
@@ -255,32 +298,39 @@ local.yml
         - django
     
     ...
+#. On the django service, confirm that you are connected to the ``envs`` files and remove/comment out the ``ports``. ::
+
+    ...
+    env_file:
+      - ./.envs/.local/.django
+    ...
+    # ports:
+    #  - 8000:8000
```

Comment:
I followed these instructions and needed to keep this configuration in order to get local SSL to work.